open-metadata-implementation/repository-services/repository-services-implementation/pom.xml

```diff
@@ -41,6 +41,11 @@
             <artifactId>slf4j-api</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
(Note that guava is already being used in other places in egeria -- it was just not being used in this particular project)